src/app/GitUI/CommandsDialogs/SettingsDialog/Pages/GitConfigSettingsPage.cs

```diff
@@ -21,6 +21,9 @@ public partial class GitConfigSettingsPage : GitConfigBaseSettingsPage
     private readonly GitConfigSettingsPageController _controller;
     private DiffMergeToolConfigurationManager? _diffMergeToolConfigurationManager;
 
+    [GeneratedRegex(@"\$(LOCAL|REMOTE|BASE|MERGED)")]
+    private static partial Regex WslRebaseRegex();
```

Comment:
Latest with C#14, this can be turned into a property in order to get rid of the confusing function call on usage, i.e.:

```suggestion
    private static partial Regex WslRebaseRegex { get; }
```